test/ui-test/testSuites/suite_status/tst_createCommunity/test.feature

```diff
@@ -0,0 +1,31 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file	test.feature
+# *
+# * \test	Status Desktop - Community
+# * \date	July 2022
+# **
+# *****************************************************************************/
+
+Feature: Status Desktop community
+
+    As a user I want to create a community and chat
+
+    The following scenarios cover basic flows of creating a community
+
+    Background:
+        Given A first time user lands on the status desktop and generates new key
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        Then the user opens the community portal section
```

Comment:
Maybe we can use the same syntax than previous "lands on" when opening a new "page" and also, apart from clicking the button, verify that the new view is there (added related comment in another point)